pkg/combustion/rpm.go

```diff
@@ -22,28 +24,34 @@ var modifyRPMScript string
 func configureRPMs(ctx *image.Context) ([]string, error) {
 	rpmSourceDir, err := generateRPMPath(ctx)
 	if err != nil {
+		log.AuditComponentFailed(rpmComponentName)
 		return nil, fmt.Errorf("generating RPM path: %w", err)
 	}
 	// Only proceed with processing the RPMs if the directory exists
 	if rpmSourceDir == "" {
+		log.AuditComponentSkipped(rpmComponentName)
 		return nil, nil
 	}
 
 	rpmFileNames, err := getRPMFileNames(rpmSourceDir)
 	if err != nil {
+		log.AuditComponentFailed(rpmComponentName)
 		return nil, fmt.Errorf("getting RPM file names: %w", err)
 	}
 
 	err = copyRPMs(rpmSourceDir, ctx.CombustionDir, rpmFileNames)
 	if err != nil {
+		log.AuditComponentFailed(rpmComponentName)
 		return nil, fmt.Errorf("copying RPMs over: %w", err)
 	}
 
 	script, err := writeRPMScript(ctx, rpmFileNames)
 	if err != nil {
+		log.AuditComponentFailed(rpmComponentName)
```

Comment:
The repetition is fine for now. Maybe in the future we can extract a `logComponentStatus()` function and defer it for each component. It would check whether the function returns a non-nil error (by naming the returned value) and then either log success or failure.